sources/zafira-dbaccess/src/main/java/com/qaprosoft/zafira/dbaccess/utils/TenancyContext.java

```diff
@@ -27,7 +28,9 @@
     private static final ThreadLocal<String> tenant = new InheritableThreadLocal<>();
 
     public static void setTenantName(String tenantName) {
-        tenant.set(tenantName != null ? tenantName.toLowerCase() : null);
+        tenantName = tenantName != null ? tenantName.toLowerCase() : null;
+        tenant.set(tenantName);
+        MDC.put("tenant.name", tenantName);
```

Comment:
Super minor note: should we stick to `tenant` as a marker name? Noticed that majority of built-in marker do not use dot notation. (e.g. `%level`, `%msg`)